### DISCUSSION TITLE
[Question]: After generate photo not show

### DISCUSSION THREAD

**Discussion by ZubikIT** [danny-avila/LibreChat#3306] [Troubleshooting]
### What is your question?

<img width="1439" alt="image" src="https://github.com/danny-avila/LibreChat/assets/745561/7c9e2e63-f711-4c11-b23b-96f5eab043cc">

### More Details

In you site work
<img width="1432" alt="image" src="https://github.com/danny-avila/LibreChat/assets/745561/0673fa9a-3881-43fa-9b67-6aff97d7d548">

### What is the main subject of your question?

UI

### Screenshots

<img width="905" alt="image" src="https://github.com/danny-avila/LibreChat/assets/745561/2d51fbaf-4855-4467-a7c2-f4b39debd836">
May be not correct permissions?
But first time I create photo 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by meroo36** [comment]
What is the URL of the generated image?

**Comment by ZubikIT** [comment]
https://chagpt.by/images/668a971cb09ac796a36ba682/img-262ff3ce-f32e-4e9a-a569-78b4ebd572fa.png
